src/Umbraco.Core/EmbeddedResources/Lang/da.xml

```diff
@@ -2318,7 +2318,7 @@ Mange hilsner fra Umbraco robotten
     <key alias="FolderCreation">Mappeoprettelse</key>
     <key alias="FileWritingForPackages">Filskrivning for pakker</key>
     <key alias="FileWriting">Filskrivning</key>
-    <key alias="MediaFolderCreation">Medie mappeoprettelse</key>
+    <key alias="MediaFolderCreation">Mediemappe oprettelse</key>
```

Comment:
Again with compound words, I'd argue that this should be `Mediemappeoprettelse`.

Reply:
👍